snakemake/parser.py

```diff
@@ -611,6 +611,22 @@ def args(self):
         )
 
 
+class Subworkflow_RuleLike(Script):
```

Comment:
```suggestion
class SubworkflowRuleLike(Script):
```
Not sure what naming convention this was, but it should be upper camel case for class names

Reply:
There was actually no naming convention here :D
I was only looking for a placeholder which would not clash with the already existing `Subworkflow` class (assuming that this class would porbably be deprecated at some point).